cms/djangoapps/contentstore/views/tests/test_library.py

```diff
@@ -87,18 +113,43 @@ def test_create_library(self):
     @patch.dict('django.conf.settings.FEATURES', {'ENABLE_CREATOR_GROUP': True})
     def test_lib_create_permission(self):
         """
-        Users who are not given course creator roles should still be able to
-        create libraries.
+        Users who are given course creator roles should be able to create libraries.
         """
         self.client.logout()
         ns_user, password = self.create_non_staff_user()
         self.client.login(username=ns_user.username, password=password)
+        add_users(self.user, CourseCreatorRole(), ns_user)
+        response = self.client.ajax_post(LIBRARY_REST_URL, {
+            'org': 'org', 'library': 'lib', 'display_name': "New Library",
+        })
+        self.assertEqual(response.status_code, 200)
 
+    @patch.dict('django.conf.settings.FEATURES', {'ENABLE_CREATOR_GROUP': False})
+    def test_lib_create_permission_no_course_creator_role_and_no_course_creator_group(self):
+        """
+        Users who are not given course creator roles should still be able to create libraries if COURSE_CREATOR_GROUP is not enabled
```

Comment:
Minor issue in this docstring: `COURSE_CREATOR_GROUP` is not the flag; it's actually `ENABLE_CREATOR_GROUP`
(I think that's my fault, since you copied my example - sorry!)